### PR TITLE
Add anaconda-verify

### PR DESCRIPTION
**Pull request by jakirkham**
Add a recipe for `anaconda-verify`. Written from scratch. This could be really useful for verifying our packages post-build and possibly linting our recipes. Feedback and maintainers welcome.

cc @conda-forge/core @ilanschnell

### CODE REVIEW

**Comment by jakirkham** [comment]
Note this recipe passes `anaconda-verify` version `1.2.0`. 😄

**Comment by jakirkham** [comment]
Would be great to have a review and possibly some other maintainers. 😉

File: recipes/anaconda-verify/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  imports:
+    - anaconda_verify
+
+  commands:
+    - anaconda-verify --help
+
+about:
+  home: https://github.com/Anaconda-Platform/anaconda-client
+  license: BSD 3-Clause
+  summary: create installer from conda packages
```
Comment (pelson) [inline recipes/anaconda-verify/meta.yaml]: Woops.

File: recipes/anaconda-verify/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - anaconda_verify
+
+  commands:
+    - anaconda-verify --help
+
+about:
+  home: https://github.com/Anaconda-Platform/anaconda-client
```
Comment (pelson) [inline recipes/anaconda-verify/meta.yaml]: Woops.

**Comment by pelson** [comment]
Couple of copy&pasta problems, but otherwise, 👍 . Happy for you to self-merge this one when you have iterated.

File: recipes/anaconda-verify/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/anaconda-verify/meta.yaml]: Ironically this one came from [upstream]( https://github.com/ContinuumIO/anaconda-verify/blob/9b58cb37ef5750faf9be2ecf95fc78e7cc4e7004/setup.py#L21 ), but I agree it should be changed. See this PR ( https://github.com/ContinuumIO/anaconda-verify/pull/5 ). Though I must have forgotten to do that here.

**Comment by jakirkham** [comment]
Corrected the metadata.

File: recipes/anaconda-verify/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{% set name = "anaconda-verify" %}
+{% set version = "1.2.0" %}
+{% set checksum = "e790262f88b3c3ef4ac9d647e0a385ea8ec665ba34509879d0a2270456f5220e" %}
+
+package:
+  name: {{ name.lower() }}
```
Comment (patricksnape) [inline recipes/anaconda-verify/meta.yaml]: Why do we need 'lower()'?

Comment (jakirkham) [inline recipes/anaconda-verify/meta.yaml]: We don't. I was trying to come up with a PyPI template and this is a holdover from it.

Comment (jakirkham) [inline recipes/anaconda-verify/meta.yaml]: This has been removed.

**Pull request merged**